src/kernel.jl

```diff
@@ -259,7 +259,7 @@ bickley() = product2d(KernelFactors.bickley())
 bickley(extended, d) = (broadcast(*, KernelFactors.bickley(extended, d)...),)
 
 """
-    gaussian((σ1, σ2, ...), [(l1, l2, ...)]) -> g
+    gaussian((σ1, σ2, ...), (l1, l2, ...)) -> g
```

Comment:
This is an undocumented manpage convention that anything listed in `[]` is optional, thus here it indicates two usage:

```julia
gaussian((σ1, σ2, ...), (l1, l2, ...)) -> g
gaussian((σ1, σ2, ...)) -> g
```

see also https://stackoverflow.com/questions/23242493/linux-unix-man-page-syntax-conventions

We should probably simplify it to `gaussian(σ, [l])` and document that both `σ` and `l` can be a tuple.